css-canvas-medical/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-fourth align-items-center justify-content-flex-start padding-l-50">
+        <img src="images/logo-blue.svg" alt="blue logo" class="blue-logo-image-size">
+      </div>
+      <div class="column-fourth"></div>
```

Comment:
You should never have empty `column` elements. Instead, you should make the previous `column-fourth` into a `column-half`